src/main/java/com/gotocompany/firehose/sink/grpc/GrpcSinkFactory.java

```diff
@@ -21,23 +31,64 @@
  * using the configurations supplied and invoke {@see #create(Map < String, String > configuration, StatsDClient client)}
  * to obtain the GrpcSink sink implementation.
  */
+@Slf4j
 public class GrpcSinkFactory {
 
+    private static final String CERT_DIRECTORY = "/usr/local/share/ca-certificates";
+
+    private static File createCertFileFromBase64(String base64Cert) throws IOException {
+        byte[] decodedBytes = Base64.getDecoder().decode(base64Cert);
+        File certDirectory = new File(CERT_DIRECTORY);
+        if (!certDirectory.exists()) {
+            certDirectory.mkdirs();
+        }
+        File certFile = Files.createTempFile(certDirectory.toPath(), "goto-root-ca", ".crt").toFile();
```

Comment:
Can we have a generic name for the cert file if it does not break things.
`goto-root-ca` -> `root-ca` or `client-cert`